internal/access/signup.go

```diff
@@ -13,38 +15,39 @@ import (
 
 // Signup creates a user identity using the supplied name and password and
 // grants the identity "admin" access to Infra.
-func Signup(c *gin.Context, name, password, orgName string) (*models.Identity, *models.Organization, error) {
+func Signup(c *gin.Context, keyExpiresAt time.Time, name, password string, org *models.Organization) (*models.Identity, string, error) {
 	// no authorization is setup yet
 	db := getDB(c)
 
-	org := &models.Organization{Name: orgName}
+	err := checkPasswordRequirements(db, password)
+	if err != nil {
+		return nil, "", err
+	}
+
 	org.SetDefaultDomain()
+	// lower-case domains look better
+	org.Domain = strings.ToLower(org.Domain)
```

Comment:
I think it would be nice to do this in `SetDefaultDomain` so that the creating the domain values is all done in the same place.